zeeguu/core/word_scheduling/basicSR/basicSR.py

```diff
@@ -116,9 +143,63 @@ def find_or_create(cls, db_session, bookmark):
         db_session.commit()
         return b
 
+    @classmethod
+    def priority_bookmarks_to_study(cls, user, required_count):
+        """
+        Prioritizes the bookmarks to study. To randomize the
+        exercise order utilize the Frontend assignBookmarksToExercises.js
+
+        The original logic is kept in bookmarks_to_study as it is called to
+        get similar_words to function as distractors in the exercises.
+
+        Currently, we prioritize bookmarks in the following way:
+        1. Words that are closest to being learned (indicated by `cooling_interval`, the highest the closest it is)
+        2. Words that are most common in the language (utilizing the word rank in the db)
+        """
+
+        def sorting_properties(bookmark):
```

Comment:
PyCharm suggestion: 

![image](https://github.com/zeeguu/api/assets/464519/6f476176-45f1-4d4b-afe8-386ebb7d041e)

Reply:
I am not sure what the comment of "Shadows name", is it because we have a class called Bookmark?